platformio.ini

```diff
@@ -40,26 +40,27 @@ lib_deps =
     DallasTemperature
 
 [env:esp32]
-platform = espressif32
+platform = https://github.com/platformio/platform-espressif32.git#feature/stage
 framework = arduino
 board = lolin32
 monitor_speed = 115200
 board_build.extra_flags = -DARDUINO_ESP32_DEV -DBOARD_HAS_PSRAM -mfix-esp32-psram-cache-issue
 board_upload.flash_size = 16MB
 board_upload.maximum_size = 16777216
+board_build.partitions = default_16MB.csv
 lib_deps =
     ReactESP
     https://github.com/tzapu/WiFiManager#bbe984c93b71601aad9f50f765ba9a255478bff3
     ESPmDNS
     marian-craciunescu/uSSDP-ESP32
     ArduinoJson
     WebSockets
-    ESPAsyncTCP
-    ESP Async WebServer
+    me-no-dev/AsyncTCP#6fab7724abe9fb1669a77e8a346d12ec148ff19a
```

Comment:
Here, too: if AsyncTCP requires some branch to work, pin to that branch, not to a single commit.

Reply:
Changed to Branch as should be ok :)